pages/jobs/index.js

```diff
@@ -74,42 +50,70 @@ const hasPosts = Object.keys(posts).length;
+              <span role="img">💡</span> Want to post a job offer?{" "}
+              <a
+                className="ml-1 text-primary-500 hover:text-primary-400"
+                href={`https://github.com/asyncapi/website/new/master/pages/jobs?value=${jobPostUrl}`}
+                target="_blank"
+              >
+                Post it now!
+              </a>
             </p>
-            <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400"> Do you want to discuss your job offer first?
-            <a className="ml-1 text-primary-500 hover:text-primary-400" href="https://github.com/asyncapi/website/issues/new" target="_blank">Get started here.</a>
+            <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400">
+              {" "}
+              Do you want to discuss your job offer first?
+              <a
+                className="ml-1 text-primary-500 hover:text-primary-400"
+                href="https://github.com/asyncapi/website/issues/new"
+                target="_blank"
+              >
+                Get started here.
+              </a>
             </p>
             <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400">
               We have an <img className="ml-1 text-primary-500 hover:text-primary-400" style={{ display: 'inline' }} src="/img/logos/rss.svg" height="18px" width="18px" /> <a className="ml-1 text-primary-500 hover:text-primary-400" href="jobs/rss.xml">RSS Feed</a> too!
             </p>
           </div>
           <div className="text-center">
-            {!hasPosts ? <div className="flex content-center justify-center">
-             <div>
-             <Empty/>
-             <p className="mt-3 max-w-2xl mx-auto text-xl leading-7 text-gray-500">No open positions currently. Check back later</p>
-             </div>
-            </div> : <div className="mt-8 flex flex-col items-stretch sm:rounded-md text-left">
-              <div className="divide-y divide-gray-200 sm:w-2/3 sm:self-center">
-                <Filter data={navItems} onFilter={onFilter} />
+            {!hasPosts ? (
+              <div className="flex content-center justify-center">
+                <div>
+                  <Empty />
+                  <p className="mt-3 max-w-2xl mx-auto text-xl leading-7 text-gray-500">
+                    No open positions currently. Check back later
+                  </p>
+                </div>
+              </div>
+            ) : (
+              <div className="mt-8 flex flex-col items-stretch sm:rounded-md text-left">
+                <div className="divide-y divide-gray-200 mb-2 sm:w-2/3 sm:self-center">
+                  <Filter
+                    className="w-full inline-flex mx-px justify-center sm:mt-0 sm:w-1/5 sm:text-sm"
+                    data={navItems}
+                    onFilter={onFilter}
+                    filter={filter}
+                    setFilter={setFilter}
```

Comment:
Filter hasn't `filter` and `setFilter` props. Why you defined them? It's an old code, or what?